Update MacOS Dependencies part

1. Torch is missed in the document, we can install it with

        pip3 install torch torchvision

2. As [Scripy's document](https://www.scipy.org/install.html) said:

    > Homebrew no longer packages NumPy or other packages of the scientific Python stack. If you’re a Homebrew user, install Python3 with Homebrew and the rest with pip:

    We should use 

        pip install scipy

    instead of 

        brew install scipy
    
3. After we install opencv, we have to do extra setup so we can 'import cv2' correctly, I'm not sure if we document about it. like [this article suggest](https://www.pyimagesearch.com/2016/12/19/install-opencv-3-on-macos-with-homebrew-the-easy-way/)

    $ echo /usr/local/opt/opencv3/lib/python3.7/site-packages >> /usr/local/lib/python3.7/site-packages/opencv.pth

I can create a PR for this later.

I had similar problems setting up Pupil v1.10 on macOS Mojave, which are not mentioned in the guide:

1.  _Problem:_ `/Users/mb/git/libuvc/include/libuvc/libuvc_internal.h:15:10: fatal error: 'libusb-1.0/libusb.h' file not found
#include <libusb-1.0/libusb.h>`

_Solution:_ https://github.com/tolga9009/elgato-gchd/issues/22

2. _Problem:_ `MainProcess - [INFO] os_utils: Disabled idle sleep.
world - [ERROR] launchables.world: Process Capture crashed with trace:
Traceback (most recent call last):
  File "/Users/mb/git/pupil/pupil_src/launchables/world.py", line 122, in world
    from file_methods import Persistent_Dict
  File "/Users/mb/git/pupil/pupil_src/shared_modules/file_methods.py", line 25, in <module>
    ), "msgpack out of date, please upgrade to version (0, 5, 6 ) or later."
AssertionError: msgpack out of date, please upgrade to version (0, 5, 6 ) or later.`

_Solution:_ pip3 install msgpack==0.5.6

3. _Problem:_ Traceback (most recent call last):
  File "/Users/mb/git/pupil/pupil_src/launchables/world.py", line 143, in world
    from plugin_manager import Plugin_Manager
  File "/Users/mb/git/pupil/pupil_src/shared_modules/plugin_manager.py", line 14, in <module>
    from calibration_routines import Calibration_Plugin, Gaze_Mapping_Plugin
  File "/Users/mb/git/pupil/pupil_src/shared_modules/calibration_routines/__init__.py", line 15, in <module>
    from .fingertip_calibration import Fingertip_Calibration
  File "/Users/mb/git/pupil/pupil_src/shared_modules/calibration_routines/fingertip_calibration/__init__.py", line 12, in <module>
    from calibration_routines.fingertip_calibration.fingertip_calibration import (
  File "/Users/mb/git/pupil/pupil_src/shared_modules/calibration_routines/fingertip_calibration/fingertip_calibration.py", line 13, in <module>
    import torch
ModuleNotFoundError: No module named 'torch'

_Solution:_ pip3 install torch

And finally your #2.

--- a/developer-docs/macos.md
+++ b/developer-docs/macos.md
@@ -43,7 +43,6 @@ Let's get started! Its time to put `brew` to work! Just copy paste commands into
 
 ```
 brew install pkg-config
-brew install scipy
 brew install libjpeg-turbo
 brew install libusb
 brew install portaudio
@@ -82,6 +81,7 @@ sudo make install
 PyOpenGL, ZMQ, ...
 
 ```
+pip3 install scipy
 pip3 install PyOpenGL
 pip3 install pyzmq
 pip3 install numexpr

